WooCommerce/Classes/ViewRelated/InAppFeedback/InAppFeedbackCardViewController.xib

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="16097.2" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="16087"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="InAppFeedbackCardViewController" customModule="WooCommerce" customModuleProvider="target">
+            <connections>
+                <outlet property="didNotLikeButton" destination="fwj-Sy-QD3" id="lHh-PH-TAX"/>
+                <outlet property="likeButton" destination="Sgc-1W-ZK6" id="qRZ-w4-Hw2"/>
+                <outlet property="titleLabel" destination="knB-Iy-ibQ" id="mYE-x1-nqG"/>
+                <outlet property="view" destination="iN0-l3-epB" id="NfP-gC-jAJ"/>
+            </connections>
+        </placeholder>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB">
+            <rect key="frame" x="0.0" y="0.0" width="320" height="140"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <stackView opaque="NO" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" axis="vertical" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="ffG-Ca-TIf" userLabel="VerticalStackView">
+                    <rect key="frame" x="15" y="16" width="290.5" height="108"/>
+                    <subviews>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Enjoying the WooCommerce app?" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="knB-Iy-ibQ">
+                            <rect key="frame" x="16" y="0.0" width="258.5" height="20.5"/>
+                            <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                            <nil key="textColor"/>
+                            <nil key="highlightedColor"/>
+                        </label>
+                        <stackView opaque="NO" contentMode="scaleToFill" distribution="fillEqually" spacing="15" translatesAutoresizingMaskIntoConstraints="NO" id="nuf-l1-QC2" userLabel="ButtonsStackView">
```

Comment:
nit: I was looking at how the buttons are aligned and saw that the configuration is in the xib. do you think these stack view customizations could be in the code? so that if we ever move to SwiftUI, we can map the key parameters right away since the stack view in the xib has so many properties.

Reply:
That makes sense! Thanks for pointing it out. Fixed in c11a1f054c4314a551b25891db52c2f9a40703d7.